Mathlib/RingTheory/DedekindDomain/Basic.lean

```diff
@@ -103,9 +133,18 @@ This is the default implementation, but there are equivalent definitions,
 TODO: Prove that these are actually equivalent definitions.
 -/
 class IsDedekindDomain
-  extends IsDomain A, IsNoetherian A A, DimensionLEOne A, IsIntegrallyClosed A : Prop
+  extends IsDomain A, IsDedekindRing A : Prop
 #align is_dedekind_domain IsDedekindDomain
 
+/-- A Dedekind domain is exactly a Dedekind ring with an extra `IsDomain` hypothesis.
```

Comment:
This docstring already appears above. Do we want it twice (I have no special opinion on that, just to check this is not an accident)?.

Reply:
It is more or less on purpose: I like to start docstrings with a description of the declaration itself, before going to implementation details. How about the following:

```suggestion
/-- Make a Dedekind domain from a Dedekind ring given that it is a domain.
```